content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -201,34 +221,33 @@ allow.textmode=true
 how.nice.to.look=fairlyNice
 ```
 
-Use the option `--from-env-file` to create a ConfigMap from an env-file, for example:
+Use the option `--from-env-file` to create a `ConfigMap` from an env-file.
+env-files contain a list of environment variables.
```

Comment:
Could replace, "env-files" with something like:
"An environment variables file or _env-file_ contains a list of environment variables."